packages/application-shell-connectors/src/components/application-context/application-context.spec.tsx

```diff
@@ -43,6 +45,8 @@ describe('rendering', () => {
             env: 'development',
             cdnUrl: 'http://localhost:3001',
             servedByProxy: false,
+            // extra props
+            foo: 'bar',
```

Comment:
Just to test that the `AdditionalEnvironmentProps` works

<img width="1193" alt="image" src="https://user-images.githubusercontent.com/1110551/59389897-ec7cd100-8d6f-11e9-93cc-90bd6d15da71.png">